packages/ui-buttons/src/Button/__new-tests__/Button.test.tsx

```diff
@@ -74,7 +74,7 @@ describe('<Button/>', () => {
     expect(button).toHaveTextContent(children)
   })
 
-  it('should render a button', () => {
+  it('should render a button with name prop', () => {
```

Comment:
`Button` does not have a `name` property. The text content `Hello World` is its child.

Reply:
good catch, I'll delete it